<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Isochrone_ING_API</class>
 <widget class="QDialog" name="Isochrone_ING_API">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>100</horstretch>
    <verstretch>100</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>880</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>1363</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Isochrone IGN API</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_input">
     <item row="0" column="0">
      <widget class="QLabel" name="label_layer_QGIS">
       <property name="text">
        <string>Vector layer</string>
       </property>
       <property name="buddy">
        <cstring>comboBox_layer_QGIS</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="comboBox_layer_QGIS">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentText">
        <string/>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
       <property name="placeholderText">
        <string>-- Select your vector layer --</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_processingMode">
       <item>
        <property name="text">
         <string>Merge isochrones by cost type</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Separate each isochron</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Merge isochrones by cost type and intersect them by Vorono√Ø polygons</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_processingMode">
       <property name="text">
        <string>Processing mode</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_isochrone_parameters">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item row="4" column="2">
      <widget class="QComboBox" name="comboBox_Unit">
       <item>
        <property name="text">
         <string>minute</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>second</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>hour</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>standard</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_CostValue">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="placeholderText">
        <string>5, 10 ,15, 20 ...</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="comboBox_profile">
       <item>
        <property name="text">
         <string>car</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>pedestrian</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QComboBox" name="comboBox_CostType">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>time</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>distance</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_CostType">
       <property name="text">
        <string>Cost type</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label_fonction_exterieur">
       <property name="text">
        <string>Location type</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QComboBox" name="comboBox_location_type">
       <item>
        <property name="text">
         <string>arrival</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>departure</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_profile">
       <property name="text">
        <string>Displacement mode</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_resource">
       <property name="text">
        <string>Resource (be aware that each resource works differently)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_Distance_exterieur">
       <property name="text">
        <string>Cost unit</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QComboBox" name="comboBox_resource">
       <item>
        <property name="text">
         <string>bdtopo-valhalla</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>bdtopo-pgr</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_CostValue">
       <property name="text">
        <string>Cost value</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QComboBox" name="comboBox_key">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="currentText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="comboBox_key_label">
       <property name="text">
        <string>Add key layer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_constraint">
     <property name="text">
      <string>Constraints used for calculation (must be a JSON object)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_constraint">
     <property name="placeholderText">
      <string>Ex: {&quot;constraintType&quot;:&quot;banned&quot;,&quot;key&quot;:&quot;wayType&quot;,&quot;operator&quot;:&quot;=&quot;,&quot;value&quot;:&quot;autoroute&quot;}</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_QTableWidget_buttons">
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_Ajouter_table">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>700</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_Effacer_ligne_table">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerItem</enum>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>39</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>123</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>30</number>
     </attribute>
     <attribute name="verticalHeaderShowSortIndicator" stdset="0">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Vector layer</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cost value</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Processing mode</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Parameters</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="locale">
      <locale language="English" country="UnitedStates"/>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>Isochrone_ING_API</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>9</x>
     <y>568</y>
    </hint>
    <hint type="destinationlabel">
     <x>820</x>
     <y>560</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>Isochrone_ING_API</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>9</x>
     <y>568</y>
    </hint>
    <hint type="destinationlabel">
     <x>742</x>
     <y>560</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
